tasks/main.yml

```diff
@@ -38,10 +38,14 @@
   notify: Restart Caddy
 
 - name: Create directories
-  file: path={{ item }} state=directory owner={{ caddy_user }} mode=0700
+  file: path={{ item.dir }} state=directory owner={{ item.user|default(caddy_user) }} group={{ item.group|default('root') }} mode={{ item.mode|default(750) }}
```

Comment:
Why not group caddy_user? Why complicate everything with defaults? Can't you just add `"{{ caddy_log_dir }}"` to `with_items`?